core/src/main/scala/org/broadinstitute/dsde/rawls/model/SamModel.scala

```diff
@@ -105,6 +110,11 @@ object SamBillingProjectPolicyNames {
   val canComputeUser = SamResourcePolicyName("can-compute-user")
 }
 
+object SamGoogleProjectPolicyNames {
+  val owner = SamResourcePolicyName("owner")
+  val notebookUser = SamResourcePolicyName("notebook-user")
+}
+
```

Comment:
we should not be adding any policies to google project resources, all policies should be inherited from the parent

Reply:
that makes sense! thanks!